SayTheirNames/Source/Model/Donation.swift

```diff
@@ -29,11 +29,13 @@ public struct Donation: Decodable {
     let title: String
     let description: String
     let link: String
-    let person: Person
-    let type: DonationType
+    let person: Person?
+    let type: DonationType?
+    let bannerImagePath: String?
```

Comment:
Do we want these are optional?

Reply:
I dont think we want them as optional, but the current data set is providing null for some objects and essentially breaking the whole screen when there isnt a value for one object

Reply:
don't see an issue if the api is returning these as null?